pkg/disk/partition_table.go

```diff
@@ -188,6 +188,9 @@ func mkESP(size uint64) Partition {
 }
 
 type CustomPartitionTableOptions struct {
+	// PartitionTableType must be either "dos" or "gpt". Defaults to "gpt".
+	PartitionTableType string
```

Comment:
We have an enum-like type for filesystem types, I think it would make sense to be consistent and do the same here instead of strings.

Reply:
Yeah, good idea.

Reply:
Done.